independent-publisher-connectors/PowerBI/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1062 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Default title",
+    "description": "Custom connector that connects to the Power BI Admin API (https://docs.microsoft.com/en-us/rest/api/power-bi/admin)",
+    "version": "1.0",
+	"contact" : {
+		"name" : "Paul Culmsee",
+		"url" : "https://docs.microsoft.com/en-us/power-bi/fundamentals/power-bi-overview",
+		"email" : "paul.culmsee@sevensigma.com.au"
+	}
+  },
+  "host": "api.powerbi.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/v1.0/myorg/admin/groups": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "@odata.context": {
+                  "type": "string",
+                  "description": "@odata.context"
+                },
+                "@odata.count": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "@odata.count"
+                },
+                "value": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "string",
+                        "description": "id"
+                      },
+                      "isReadOnly": {
+                        "type": "boolean",
+                        "description": "isReadOnly"
+                      },
+                      "isOnDedicatedCapacity": {
+                        "type": "boolean",
+                        "description": "isOnDedicatedCapacity"

```

Comment:
@paulculmsee Can you please provide valid description values for all the properties in swagger ?

Documentation - [Description values](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#operation-and-parameter-summaries-and-descriptions)

Reply:
done